java-generator/core/src/main/java/io/fabric8/java/generator/WritableCRCompilationUnit.java

```diff
@@ -28,44 +27,35 @@
 
 public class WritableCRCompilationUnit {
 
-    private final CompilationUnit cu;
-    private final List<String> classNames;
+    private final List<GeneratorResult.ClassResult> classResults;
 
-    WritableCRCompilationUnit(CompilationUnit cu, List<String> classNames) {
-        this.cu = cu;
-        this.classNames = classNames;
+    public List<GeneratorResult.ClassResult> getClassResults() {
```

Comment:
This is just about coding style & me, but I'd keep this property after the ctor definition.

Reply:
done